mlrun/projects/project.py

```diff
@@ -206,6 +206,7 @@ def load_project(
     subpath: str = None,
     clone: bool = False,
     user_project: bool = False,
+    sync_functions: bool = False,
```

Comment:
I think it's better to add the new param (`sync_functions`) last, after `save`, for not breaking code of users that don't pass parameters by name.